tests/combiners_test.py

```diff
@@ -924,63 +926,54 @@ def create_combiner(return_values: List[bool]):
-        aggregate_params = _create_aggregate_params(vector_size=vector_size)
+        aggregate_params = _create_aggregate_params(vector_size=vector_size, vector_norm_kind=vector_norm_kind,
+                                                    vector_max_norm=vector_max_norm)
         params = dp_combiners.CombinerParams(mechanism_spec, aggregate_params)
         return dp_combiners.VectorSumCombiner(params)
 
-    @parameterized.named_parameters(
-        dict(testcase_name='no_noise', no_noise=True),
-        dict(testcase_name='noise', no_noise=False),
+    @parameterized.product(
+        testcase=[dict(input_vector=[[]], output_vector=[]),
+                  dict(input_vector=[[0]], output_vector=[0]),
+                  dict(input_vector=[[0, 0]], output_vector=[0, 0]),
+                  dict(input_vector=[[1], [2]], output_vector=[3]),
+                  dict(input_vector=[[1, 2], [3, 4]], output_vector=[4, 6]),
+                  dict(input_vector=[[1, 2, 3], [4, 5, 6]], output_vector=[5, 7, 9])],
+        norm_kind=[NormKind.Linf, NormKind.L1, NormKind.L2]
     )
-    def test_create_accumulator(self, no_noise):
-        combiner = self._create_combiner(no_noise, vector_size=1)
-        self.assertEqual(np.array([0.]), combiner.create_accumulator([[0.]]))
-        self.assertEqual(
-            np.array([2.]),
-            combiner.create_accumulator([np.array([1.]),
-                                         np.array([1.])]))
-
-    @parameterized.named_parameters(
-        dict(testcase_name='Linf_no_noise',
-             norm_kind=NormKind.Linf,
-             no_noise=True),
-        dict(testcase_name='Linf_noise',
-             norm_kind=NormKind.Linf,
-             no_noise=False),
+    def test_create_accumulator_sums_correctly(self, testcase, norm_kind):
+        combiner = self._create_combiner(no_noise=True, vector_size=len(testcase['input_vector'][0]),
+                                         vector_norm_kind=norm_kind,
+                                         vector_max_norm=999)  # no clipping
+        self.assertIsNone(np.testing.assert_array_equal(
+            combiner.create_accumulator(testcase['input_vector']), testcase['output_vector']))
+
+    @parameterized.parameters(
+        dict(norm_kind=NormKind.Linf, norm=5, input_vector=[[6]], output_vector=[5]),
+        dict(norm_kind=NormKind.Linf, norm=5, input_vector=[[6], [7]], output_vector=[5]),
+        dict(norm_kind=NormKind.Linf, norm=5, input_vector=[[5, 6]], output_vector=[5, 5]),
         # L0 noise not implemented at the moment
-        dict(testcase_name='L1_no_noise', norm_kind=NormKind.L1, no_noise=True),
-        dict(testcase_name='L1_noise', norm_kind=NormKind.L1, no_noise=False),
-        dict(testcase_name='L2_no_noise', norm_kind=NormKind.L2, no_noise=True),
-        dict(testcase_name='L2_noise', norm_kind=NormKind.L2, no_noise=False),
+        dict(norm_kind=NormKind.L1, norm=10, input_vector=[[11]], output_vector=[10]),
```

Comment:
norm? we won't implement L0 norm, should we delete the comment at the line above?